packages/app/lib/apps.dart

```diff
@@ -9,12 +7,10 @@ import 'package:shared_preferences/shared_preferences.dart';
 
 List<AppImplementation> getAppImplementations(
   final SharedPreferences sharedPreferences,
-  final RequestManager requestManager,
-  final NeonPlatform platform,
 ) =>
     [
-      FilesApp(sharedPreferences, requestManager, platform),
-      NewsApp(sharedPreferences, requestManager, platform),
-      NotesApp(sharedPreferences, requestManager, platform),
-      NotificationsApp(sharedPreferences, requestManager, platform),
+      FilesApp(sharedPreferences),
+      NewsApp(sharedPreferences),
+      NotesApp(sharedPreferences),
+      NotificationsApp(sharedPreferences),
```

Comment:
Maybe as a followup and but once rebased I'd love to also remove the sharedPrefs from here.

Especially as we use the same one throughout the app a static field somewhere would also work (maybe in the NeonPlatform).

Reply:
Let's do this as a follow up